04-HTML5-APIs/js/drag-and-drop.js

```diff
@@ -0,0 +1,31 @@
+let drop = document.getElementById("drop");
+
+drop.ondragenter = function(event) {
+  document.getElementById("drop").textContent = '';
```

Comment:
You already declare drop (`document.getElementById("drop")`) above. You can reuse it.